src/styles.scss

```diff
@@ -1,5 +1,6 @@
 /* You can add global styles to this file, and also import other style files */
-@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro');
+@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro:200,400');
+@import url('https://fonts.googleapis.com/css?family=Biryani:200,300,400,600,900');
```

Comment:
If we're only using font weight 200, we don't need to pull in the rest of the weights unless they're used implicitly (perhaps h2?) -- but this is OK, we can sort it out with a few more edits/tweaks.